cloudify_agent/shell/commands/daemons.py

```diff
@@ -382,3 +418,35 @@ def _parse_custom_options(options):
         parsed[key] = value
 
     return parsed
+
+
+def _create_rest_ssl_cert(agent):
+    """
+    put the REST SSL cert into a file for clients to use,
+    if rest_cert_content is set.
+    """
+    click.echo('Deploying REST SSL certificate (if defined).')
+    rest_cert_content = agent['rest_cert_content']
+    if rest_cert_content and agent['verify_rest_certificate']:
```

Comment:
verify_rest_certificate has not be normalized. "True" and "False" strings are both truthy values.
The only case this will return false, is when no one supplied this value, which is irrelevant for the main path.